src/components/selection-panel/selection-panel.spec.ts

```diff
@@ -2,6 +2,7 @@ import { expect, fixture } from '@open-wc/testing';
 import { html } from 'lit/static-html.js';
 
 import { waitForLitRender } from '../core/testing';
+import { testA11yTreeSnapshot } from '../core/testing/a11y-tree-snapshot';
```

Comment:
I haven't added the `testA11yTreeSnapshot` to the `core/testing/index` file because we don't want the `testA11yTreeSnapshot` dependencies to be included in the `core` esModule.
@kyubisation Does it make sense?

Reply:
Yes, I agree. 👍